.github/workflows/chaos.yml

```diff
@@ -0,0 +1,61 @@
+name: Chaos Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: setup go
+      uses: actions/setup-go@v2.1.3
+      with:
+        go-version: '1.13'
+
+    - name: Creating kind cluster
+      uses: helm/kind-action@v1.0.0-rc.1
```

Comment:
Does these non-official and non-Apache actions can be used now?

Reply:
Thanks for the remind! Changed to a minikube startup script